src/main/java/org/broadinstitute/consent/http/db/DataAccessRequestDAO.java

```diff
@@ -87,6 +89,45 @@ WHERE dar.submission_date > now() - interval '1 year'
           """)
   List<DataAccessRequest> findApprovedDARsByDatasetId(@Bind("datasetId") Integer datasetId);
 
+  /**
+   * This query finds dataset ids submitted within the last year on dar-dataset combinations where
+   * the most recent vote is true. The query accomplishes this by creating a view that is a grouping
+   * of election reference ids and LAST vote in the group of final votes for all data access
+   * elections. We need to group them due to the case of multiple elections on a dar-dataset
+   * request. Election 1 may have been denied. Election 2 may have been approved. Election 3 may
+   * have been denied again. When we partition over the election reference id, we'll get all final
+   * votes. The `LAST_VALUE` function selects the last result in the partition, which would be
+   * `FALSE` in the example above. Outside the JOIN, we filter on groupings where the final vote
+   * value is `TRUE` so the denied election in the example would be filtered out.
+   *
+   * @param darReferenceId The DAR reference UUID
+   * @return List of approved Datasets for the DAR
+   */
+  @SqlQuery(
+      """
+              SELECT DISTINCT dd.dataset_id
```

Comment:
In theory, we shouldn't need distinct here since a dar can only reference a dataset once.